xcm/src/v3/multiasset.rs

```diff
@@ -406,7 +406,7 @@ pub struct MultiAsset {
 	/// The overall asset identity (aka *class*, in the case of a non-fungible).
 	pub id: AssetId,
 	/// The fungibility of the asset, which contains either the amount (in the case of a fungible
-	/// asset) or the *instance ID`, the secondary asset identifier.
+	/// asset) or the `instance ID`, the secondary asset identifier.
```

Comment:
I think the error was the other way.
```suggestion
	/// asset) or the *instance ID*, the secondary asset identifier.
```